.github/workflows/automerge.yaml

```diff
@@ -0,0 +1,29 @@
+name: Automerge
+
+on:
+  push:
+    branches:
+      - '!master'
+      - '!dev-branch'
+
+env:
+  # path to the branch you're automerging into.
+  AUTOMERGE_INTO: https://github.com/th3w1zard1/adaptive-lighting/tree/dev-branch.git
```

Comment:
This is your fork

Reply:
My mistake keeping my workflow separate from forks is a problem I haven't managed to fix yet. Removed in the last commit `708677e`